test/jdk/performance/client/RenderPerfTest/src/renderperf/RenderPerfTest.java

```diff
@@ -281,20 +456,18 @@ static class LargeTextParticleRenderer extends TextParticleRenderer {
         public void render(Graphics2D g2d, int id, float[] x, float[] y, float[] vx, float[] vy) {
             setPaint(g2d, id);
             if (id % 100 != 0) return;
-            Font font = new Font("LucidaGrande", Font.PLAIN, 32);
+            if (font == null) {
+                font = new Font("LucidaGrande", Font.PLAIN, 32);
```

Comment:
I see this is pre-existing, but we should change this.
That's a font available only on macOS. You should either look up a font to use on the system at run time,
or use a logical font.

Reply:
Of course, I will make constant Font instances and use family name (Dialog or SansSerif), what do you prefer?

Reply:
either dialog or sansserif. Dialog is the default font so perhaps that.

Reply:
Dialog by default.
After discussion, it may be useful to have new cli args:
--font=name --fontSize=... --text=""

Special fonts with ligature or complex cases could then be tested...